examples/README.md

```diff
@@ -1,13 +1,51 @@
 # Example: Provision an SDDC
 
-This is an example that demonstrates SDDC management actions like creating, updating and deleting an existing SDDC.
+This is an example that demonstrates the following workflow:
 
-To run the example:
+1. SDDC management actions like creating, updating and deleting an existing SDDC.
+2. Public IP management actions like creation and deletion.
+3. Site recovery management actions like activation and deactivation.
+4. SRM node management actions like creation and deletion.
+
+For this workflow, a 10 minute delay must be added after SDDC is created and before site recovery can be activated.
```

Comment:
For which workflow? 1, 2, 3 or 4? Each of the workflow steps seem to be an individual workflow by itself.

Reply:
The main.tf in the examples folder is for all the steps 1,2,3 and 4 combined. That is why we need to add the provisioner local exec which adds the 10 minute delay which is mentioned on line 10. 
I have created a separate PR for each individual workflows too.

Reply:
My point is that we don’t do all the actions you described in each of the steps. The workflow is more like create sddc, create public up, enable site recovery and finally SRM node creation.

Reply:
Agreed but terraform apply would perform creation on the resources but with a simple terraform destroy you can delete the resources too. Or in the same main.tf if someone were to update some field on any of the resources that would do an update action. Hence, I have listed all of them.

Reply:
Done